chameleonultragui/lib/helpers/mifare_classic/general.dart

```diff
@@ -156,7 +156,9 @@ MifareClassicType mfClassicGetCardTypeByBlockCount(int blockCount) {
 }
 
 int mfClassicGetSectorTrailerBlockBySector(int sector) {
-  if (sector < 32) {
+  if (sector == 0) {
```

Comment:
why add that?

Reply:
my fault, forgot to remove this. i was convinced this gave an bug to my added code